scripts/check-each-commit.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
+if [ "$1" = "" ]; then
+	echo "USAGE: $0 remote/head_branch"
+	echo "eg $0 upstream/main"
```

Comment:
nit: Should say `upstream/master` or are we planning on switching to the new branch naming system in the foreseeable future?